src/TreeHouse/IoBundle/Command/ImportRotateCommand.php

```diff
@@ -74,5 +85,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
             $this->importRotator->rotate($feed, $input->getOption('rotations'));
         }
+
+        $lockHandler->release();
```

Comment:
kan je de release niet beter in een `finally` zetten (try/catch om hele code heen) omdat eventuele exceptions er voor gaan zorgen dat de lock erop blijft zitten?